pkg/config/config_template.yaml

```diff
@@ -65,6 +65,12 @@ api_key:
 #   - environment:dev
 #   - <TAG_KEY>:<TAG_VALUE>
 
+## @param end  - string - optional
+## The environment name were the agent is running. Attached in-app to every
+## metric, event, log, trace, and service check emitted by this Agent.
+#
+# env: <environment name>
```

Comment:
```suggestion
# env: <ENVIRONMENT_NAME>
```

Reply:
Shouldn't we set a default `dev` or `test` value ?

Reply:
(the default should be empty, i.e. if the user doesn't specify anything, the agent should not send the `env:` host tag)